src/domain/experiment.py

```diff
@@ -207,29 +226,121 @@ class OtherLanguagesRandomCutoffSeedDictionaryAblationExperiment(RandomCutoffSee
         'source_language': ['en'],
         'target_language': ['et', 'fa', 'lv', 'vi']
     }
+
+    def __init__(self, base_config):
+        super().__init__(base_config)
+
+    def get_sbatch_args(self, run_params):
+        return {'time': '0-1:00'}
+
+
+class OtherLanguagesRandomSeedDictionaryAblationExperiment(OriginalExperiment):
+    EXPERIMENT_NAME = 'other_languages_random_seed_dictionary_ablation'
+    LANGUAGE_PARAMS = {'source_language': ['en'], 'target_language': ['et', 'fa', 'lv', 'vi']}
+    CHANGING_PARAMS = {'seed_dictionary_method': ['random_raw']}
+
+    def __init__(self, base_config):
+        super().__init__(base_config)
+
+    def get_sbatch_args(self, run_params):
+        return {'time': '0-1:00'}
+
+
+class OtherLanguagesRandomCutoffSeedDictionaryAblationExperiment(OriginalExperiment):
+    EXPERIMENT_NAME = 'other_languages_random_cutoff_seed_dictionary_ablation'
+    LANGUAGE_PARAMS = {'source_language': ['en'], 'target_language': ['et', 'fa', 'lv', 'vi']}
+    CHANGING_PARAMS = {'seed_dictionary_method': ['random_cutoff']}
+
+    def __init__(self, base_config):
+        super().__init__(base_config)
+
+    def get_sbatch_args(self, run_params):
+        return {'time': '0-1:00'}
+
+
+class GridSearchExperiment(OriginalExperiment):
+    def __init__(self, base_config):
+        super().__init__(base_config)
+
+    def __get_run_params(self, run):
+        if len(self.CHANGING_PARAMS) == 1:
+            return run.data.params[list(self.CHANGING_PARAMS)[0]]
```

Comment:
Why not list already `CHANGING_PARAMS`

Reply:
CHANGING_PARAMS is a dict, I only want to read the params for all changing params, where in this case it is only a single key.